src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace Microsoft.Identity.Client
 {
-    internal sealed class ApplicationConfiguration : IAppConfigInternal
```

Comment:
There is no point in having an interface over a simple data object. It's an anti-pattern.